### DISCUSSION TITLE
Error to create PDF in server

### DISCUSSION THREAD

**Discussion by Mapachosa** [thegetty/quire#432] [Development]
Hi,

A few weeks ago I translated my project to a local server. Everything went well until I wanted to create new PDFs with the corrections made on the server. We have nine publications. One of them, which is the smallest, runs the `quire pdf` command normally, but for the others it throws the following error that I attach in a txt.

It's pretty weird because it looks like it's a memory problem, but I'm not sure.

First of all, Thanks.

[ERROR.txt](https://github.com/thegetty/quire/files/9262012/ERROR.txt)

**Comment by Erin-Cecele** [comment]
Hi @Mapachosa. Wow! Nine Quire publications!! Are you able to share any of them with us? Would you like them to be featured on our [Community Showcase](https://quire.getty.edu/community/community-showcase/) or in our [newsletter](https://quire.getty.edu/community/newsletter-archive/)?

Thank you for sharing the error text. I want to make sure I'm fully understanding. Sounds like the smallest project is able to be output as a PDF without a problem, but the others, the larger projects, are receiving errors. Is that correct? Are the projects very image heavy? We will look into this for you!

**Reply by Mapachosa** [reply]
Hi Erin, 

Yes, I like to share our project with us. We are finishing some details like the PDFs and Epub.

Yes, When I run `quire pdf` for the smallest project there is not any problem. It should be said that although the other quire are larger, the images used are not heavy, but they do have a greater number of pages. However, before uploading them to the server, the `quire pdf` and `quire epub`  worked for all the projects.

In this link you see one publication of our project: http://publicaciones.mariategui.org/escritos-juveniles-tomo-i/

**Reply by Erin-Cecele** [reply]
Hi @Mapachosa, what version of Quire are you running? It should be v. 0.20.4. If you are running on an older version, please update by running the command `npm install -global @thegetty/quire-cli@0.20.4`. Let us know if that helps resolve the problem.

**Reply by Mapachosa** [reply]
Hi @Erin-Cecele 
It's been passed a long time since I wrote you.
I would like to tell you that we have already published the nines quires of our projects. Are nine websites with the possibility of downloading PDFs and Epub. We are very happy and would like to share with the Quire community.

In this link you see one publication of our project: http://publicaciones.mariategui.org and you see our nine project title "Escritos Juveniles de José Carlos Mariátegui".

Thank you very much and it has been a great experience for us.
Best,

**Reply by Erin-Cecele** [reply]
Thank you so much for sharing this with us! And thank you for completing the Quire project submission form. I received your message above as I was writing this week's newsletter which includes your 9 publications!  Keep an eye out for the newsletter later this week. We'll get them up on the Quire website soon too. 

In the meantime, congratulations on all the excellent work you have done with Quire.
